modules/monitoring-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -3,67 +3,101 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="specifying-limits-and-resource-requests-for-monitoring-components_{context}"]
-= Specifying limits and requests for monitoring components
 
-To configure CPU and memory resources, specify values for resource limits and requests in the appropriate `ConfigMap` object for the namespace in which the monitoring component is located:
+// The final solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+// tag::CPM[]
+[id="specifying-limits-and-resource-requests-for-monitoring-components-cpm_{context}"]
+= Specifying limits and requests for core monitoring components
+// end::CPM[]
 
-* The `cluster-monitoring-config` config map in the `openshift-monitoring` namespace for core platform monitoring
-* The `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace for components that monitor user-defined projects
+// tag::UWM[]
+[id="specifying-limits-and-resource-requests-for-monitoring-components-uwm_{context}"]
+= Specifying limits and requests for monitoring components for user-defined projects
+// end::UWM[]
+
+// Set attributes to distinguish between cluster monitoring example (core platform monitoring - CPM) and user workload monitoring (UWM) examples.
+// tag::CPM[]
+:configmap-name: cluster-monitoring-config
+:namespace-name: openshift-monitoring
+:alertmanager: alertmanagerMain
+:prometheus: prometheusK8s
+:thanos: thanosQuerier
+// end::CPM[]
+// tag::UWM[]
+:configmap-name: user-workload-monitoring-config
+:namespace-name: openshift-user-workload-monitoring
+:alertmanager: alertmanager
+:prometheus: prometheus
+:thanos: thanosRuler
+// end::UWM[]
+
+To configure CPU and memory resources, specify values for resource limits and requests in the `{configmap-name}` `ConfigMap` object in the `{namespace-name}` namespace.
 
 .Prerequisites
 
-* *If you are configuring core platform monitoring components*:
-** You have access to the cluster as a user with the `cluster-admin` cluster role.
-** You have created a `ConfigMap` object named `cluster-monitoring-config`.
-* *If you are configuring components that monitor user-defined projects*:
-** You have access to the cluster as a user with the `cluster-admin` cluster role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+// tag::CPM[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have created a `ConfigMap` object named `cluster-monitoring-config`.
```

Comment:
This one is appearing in the ROSA distribution. I'm not sure that ROSA/OSD users need to create this config map. For example, the user-workload-monitoring-config config map is created by default.